docs/getting-started/kubernetes-quickstart.md

```diff
@@ -0,0 +1,155 @@
+# Getting started with tracee in Kubernetes 
+
+This guide is focused on running tracee on Kubernetes. To demonstrate how to use Tracee we will use minikube, a distribution of kubernetes that can run on your local machine.
+
+## Prerequisites 
```

Comment:
Do we want to point out that their local cluster has to be on an intel processor? i.e. when people try this guide on a Mac silicon, it is not going to work

Reply:
oh you mentioned it more in 46 -- maybe move line 46 up here?